core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/autoscales/AutoscaleV4Endpoint.java

```diff
@@ -52,7 +53,14 @@
     @ApiOperation(value = StackOpDescription.PUT_BY_ID, produces = APPLICATION_JSON, notes = Notes.STACK_NOTES, nickname = "putStackForAutoscale")
     void putStack(@PathParam("crn") String crn, @PathParam("userId") String userId, @Valid UpdateStackV4Request updateRequest);
 
-    // TODO CB-14929: Define this API correctly, or remove it. Primarily used for testing. Will likely be required for the CLI.
+    // Not overloading the regular scaling API since 1) that is public, and 2) stopstart may move into a separate InstancePoolManagementController at some point
```

Comment:
Would this require an annotation for HTTP request type

Reply:
Yes it should. Will fix in the next PR, after testing the changes on a cluster.